src/screens/UserManagement/UserRevamp/UserUtils.res

```diff
@@ -0,0 +1,142 @@
+let getMerchantSelectBoxOption = (
+  ~label,
+  ~value,
+  ~dropdownList: array<OMPSwitchTypes.ompListTypes>,
+  ~showAllSelection=false,
+) => {
+  let allOptions: SelectBox.dropdownOption = {
+    label,
+    value,
+    customRowClass: "!border-b py-2",
+    textColor: "font-semibold",
+  }
+  let orgOptions =
+    dropdownList->Array.map((item): SelectBox.dropdownOption => {label: item.name, value: item.id})
+
+  if showAllSelection {
+    [allOptions, ...orgOptions]
+  } else {
+    orgOptions
+  }
+}
+
+let inviteEmail = FormRenderer.makeFieldInfo(
+  ~label="Enter email (s) ",
+  ~name="email_list",
+  ~customInput=(~input, ~placeholder as _) => {
+    let showPlaceHolder = input.value->LogicUtils.getArrayFromJson([])->Array.length === 0
+    InputFields.textTagInput(
+      ~input,
+      ~placeholder=showPlaceHolder ? "Eg: mehak.sam@wise.com, deepak.ven@wise.com" : "",
+      ~customButtonStyle="!rounded-full !px-4",
+      ~seperateByComma=true,
+    )
+  },
+  ~isRequired=true,
+)
+
+let validateEmptyValue = (key, errors) => {
+  switch key {
+  | "emailList" => Dict.set(errors, "email", "Please enter Invite mails"->JSON.Encode.string)
+  | "roleType" => Dict.set(errors, "roleType", "Please enter a role"->JSON.Encode.string)
+  | _ => Dict.set(errors, key, `Please enter a ${key->LogicUtils.snakeToTitle}`->JSON.Encode.string)
+  }
+}
+
+let validateForm = (values, ~fieldsToValidate: array<string>) => {
+  let errors = Dict.make()
+  open LogicUtils
+  let valuesDict = values->getDictFromJsonObject
+
+  fieldsToValidate->Array.forEach(key => {
+    let value = valuesDict->getJsonObjectFromDict(key)
+
+    switch value {
+    | Array(listofemails) =>
+      if listofemails->Array.length === 0 {
```

Comment:
same here

Reply:
No LogicUtils present for empty array check